galvan/src/main/java/org/terracotta/testing/master/GalvanStateInterlock.java

```diff
@@ -45,17 +44,16 @@
   // There can be one active unless we are still waiting for a server to enter an active state.
   private ServerProcess activeServer;
   // There can be any number of passives (a server enters this state when it becomes PASSIVE_STANDBY).
-  private final List<ServerProcess> passiveServers = new Vector<ServerProcess>();
+  private final List<ServerProcess> passiveServers = new ArrayList<>();
+  private final List<ServerProcess> diagnosticServers = new ArrayList<>();
   // A server which is running but hasn't yet reported in its state waits as "unknownRunning".
-  private final List<ServerProcess> unknownRunningServers = new Vector<ServerProcess>();
+  private final List<ServerProcess> unknownRunningServers = new ArrayList<>();
   // A server which has returned an exit status, or hasn't yet become running, is a terminated server.
-  private final List<ServerProcess> terminatedServers = new Vector<ServerProcess>();
-
+  private final List<ServerProcess> terminatedServers = new ArrayList<>();
 
   // ----- CLIENT STATE -----
   // When a client starts up, we register it as a running client and wait for it to report that it terminated.
-  private final List<ClientRunner> runningClients = new Vector<ClientRunner>();
-
+  private final List<ClientRunner> runningClients = new ArrayList<>();
```

Comment:
I'd leave these as Vector.  ArrayList is not synchronized.  I know many ide recommend not using vector but this is a low velocity use so Vector is still serviceable.

Reply:
All methods here are synchronized. Using a `Vector` is unnecessary, which is why I changed it to `ArrayList`